meetings/azure.yaml

```diff
@@ -2,16 +2,15 @@ project: Azure working group
 meeting_id: Azure
 project_url: https://github.com/ansible/community/wiki/azure
 agenda_url: https://github.com/ansible/community/issues?q=is:open+label:meeting_agenda+label:azure
-chair: Matt Davis (nitzmahone), Kylie Liang (kyliel)
+chair: Anshul Behl (anshulbehl), Jacob Jackson (thedoubl3j)
```

Comment:
All be it that I am fine with this for now, we will need to transition this to the respective members of the azure community, both azure devs and community members who wish to step forward and help run this.

Reply:
couldn't change it to `?` as its not allowed when running `make`

Reply:
wfm, just need to make a note to make sure that it doesn't stay with us long ;)